packages/lib-subject-viewers/src/VolumetricViewer/components/ComponentViewer.js

```diff
@@ -50,14 +52,14 @@ const StyledBox = styled(Box)`
       }
 
       .volume-controls {
-        margin-top: 20px;
+        margin-top: 0px;
```

Comment:
While most browsers will still parse this, margin or padding of `0` is usually unitless. See the examples at the top of https://developer.mozilla.org/en-US/docs/Web/CSS/margin.
```suggestion
        margin-top: 0;
```